tachyon/math/finite_fields/binary_fields/binary_fields.h

```diff
@@ -5,9 +5,60 @@
 #ifndef TACHYON_MATH_FINITE_FIELDS_BINARY_FIELDS_BINARY_FIELDS_H_
 #define TACHYON_MATH_FINITE_FIELDS_BINARY_FIELDS_BINARY_FIELDS_H_
 
+#include "tachyon/math/finite_fields/binary_fields/binary_field16_config.h"
+#include "tachyon/math/finite_fields/binary_fields/binary_field1_config.h"
+#include "tachyon/math/finite_fields/binary_fields/binary_field2_config.h"
+#include "tachyon/math/finite_fields/binary_fields/binary_field32_config.h"
+#include "tachyon/math/finite_fields/binary_fields/binary_field4_config.h"
+#include "tachyon/math/finite_fields/binary_fields/binary_field64_config.h"
+#include "tachyon/math/finite_fields/binary_fields/binary_field8_config.h"
```

Comment:
How about organizing the headers in order of 1, 2, 4, 8, 16, 32, 64 using clang-format off?